src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -225,6 +233,17 @@ public unsafe static bool TryGetPublicKey(ImmutableArray<byte> blob, out Immutab
                 // The RSA magic key must match the blob id
                 if (type == PrivateKeyBlobId && magic == RSA2)
                 {
+                    RSAParameters privKey = default;
```

Comment:
>RSAParameters [](start = 20, length = 13)

Does it matter whether the exponent array has leading 0s?

There is similar code in CoreFX: [https://source.dot.net/#System.Security.Cryptography.Csp/System/Security/Cryptography/CapiHelper.Shared.cs,160](https://source.dot.net/#System.Security.Cryptography.Csp/System/Security/Cryptography/CapiHelper.Shared.cs,160)

where they seem to avoid leading 0s for some reason.